src/app/app.component.html

```diff
@@ -15,15 +15,19 @@
         (zoomChange)="setMapZoomLevel($event)">
     </app-zoom-control>
     <div class="map-ui">
-        <app-layer-select 
-            [selectedLayer]="activeDataLevel"
-            [layers]="dataLevels" 
+        <app-ui-select
+            class="layer-select"
+            [labelProperty]="'name'"
```

Comment:
Can this (and the one below it) be simplified to `labelProperty="name"`? That worked for me locally

Reply:
Yep! Good catch, just updated.